scripts/pin_the_slow_way.py

```diff
@@ -36,6 +36,7 @@
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
           'libblitz': 'libblitz 0.10|0.10.*',
+          'libevent': 'libevent 2.0.*',
```

Comment:
I would say this is more about good practices in our pinnings rather than what can happen in reality but anyways I think this pull that I opened needs some love: https://github.com/conda-forge/conda-forge.github.io/pull/245